aws_lambda_builders/workflows/python_pip/packager.py

```diff
@@ -150,13 +150,23 @@ class DependencyBuilder(object):
     packager.
     """
 
-    _MANYLINUX_COMPATIBLE_PLATFORM = {
-        "any",
-        "linux_x86_64",
-        "manylinux1_x86_64",
-        "manylinux2010_x86_64",
-        "manylinux2014_x86_64",
+    _ADDITIONAL_COMPATIBLE_PLATFORM = {"any", "linux_x86_64"}
+    _MANYLINUX_LEGACY_MAP = {
+        "manylinux1_x86_64": "manylinux_2_5_x86_64",
+        "manylinux2010_x86_64": "manylinux_2_12_x86_64",
+        "manylinux2014_x86_64": "manylinux_2_17_x86_64",
```

Comment:
I examined the PEP 600 and `manylinux2014_x86_64` is not in there. 

<img width="823" alt="image" src="https://user-images.githubusercontent.com/3804518/118886296-998deb80-b8ad-11eb-87cb-e4b600dc22d3.png">

Reply:
`manylinux2014` is defined in [PEP 599](https://www.python.org/dev/peps/pep-0599/)